templates/signin.html

```diff
@@ -126,13 +126,13 @@ <h5>Support</h5>
           FAQ
         </a>
       </li>
-      <!--Browse-->
+      <!--Resources-->
```

Comment:
Header and footer should probably use a template so we only need to make the change once